src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -1,39 +1,32 @@
 <template>
   <div
+    class="gallery-carousel"
     :class="{
-      'gallery-carousel': true,
-      isUsingMouse,
-      withPagerInside: pagerInside,
+      'gallery-carousel--with-pager-inside': pagerInside,
     }"
-    @keydown="setUsingMouse(false)"
-    @mousedown="setUsingMouse(true)"
   >
     <div
       class="gallery-carousel__wrap"
       tabindex="0"
       @keydown.self="maybeSelectItem"
-      @blur.self="setUsingMouse(false)"
     >
       <flickity
         ref="flickity"
         :options="flickityOptions"
       >
         <div
+          class="gallery-carousel__item"
           v-for="(item, i) in items"
           :key="i"
           :class="{
-            'gallery-carousel__item': true,
-            'isActive': shouldShowActiveState(i),
+            'is-active': shouldShowActiveState(i),
           } "
         >
-          <div
-            :style="sizerStyles"
-            class="gallery-carousel__item__sizer"
-          />
+          <div :style="sizerStyles" />
 
           <!-- the item.thumbnail field is used as default value for the item image -->
           <!-- IE11 uses this value only because do not support the picture element -->
-          <picture class="gallery-carousel__item__picture">
+          <picture class="gallery-carousel__item-picture">
```

Comment:
`picture` class is not a modifier in this case. It's a block element. Are you sure about this change? 
http://getbem.com/naming/
I checked the documentation of BEM, they are saying; `You need to use Element type if the element is semantically tied to its block.`

Reply:
I searched a little bit I think we can use `gallery-carousel__picture` instead of `gallery-carousel__item__picture` to avoid using more than one element nesting.

Source: https://stackoverflow.com/a/22566593

Reply:
Another reference: https://medium.com/fed-or-dead/battling-bem-5-common-problems-and-how-to-avoid-them-5bbd23dee319

Reply:
we can go with _.gallery-carousel__picture_ even just just _.galley-carousel picture_

Reply:
Thanks a lot for the suggestion, and I totally agree with you.

Here are the changes 86be6da 
What do you thinkg?

Reply:
Bom trabalho!

![thanks](https://user-images.githubusercontent.com/14016028/91445558-901fa780-e876-11ea-9357-6a61a7fa8b46.gif)